internal/pkg/cmd/flags.go

```diff
@@ -347,7 +347,7 @@ func AddValueFormatFlag(cmd *cobra.Command) {
 }
 
 func AddByokKeyFlag(cmd *cobra.Command, command *AuthenticatedCLICommand) {
-	cmd.Flags().String("byok", "", "Confluent Cloud Key ID (CCK-ID) of a registered BYOK encryption key.")
+	cmd.Flags().String("byok", "", `Confluent Cloud Key ID (CCK-ID) of a data-at-rest encryption key (use "confluent byok create" to register a key with Confluent Cloud).`)
```

Comment:
Can you fix this for the `--region` flag as well?
```suggestion
	cmd.Flags().String("byok", "", "Confluent Cloud Key ID (CCK-ID) of a data-at-rest encryption key (use `confluent byok create` to register a key with Confluent Cloud).")
```

Reply:
Also- it would be great if you could insert these new functions in alphabetical order by flag.

Reply:
Sorted all the flags in there to restore some order.